src/platform/forms-system/src/js/components/FormSignature.jsx

```diff
@@ -114,12 +115,15 @@ export const FormSignature = ({
         required={required}
         errorMessage={(showError || signature.dirty) && signatureError}
       />
-      <Checkbox
+      <VaCheckbox
         label={checkboxLabel}
+        description={null}
         required={required}
-        onValueChange={setChecked}
-        errorMessage={showError && checkboxError}
-      />
+        error={showError && checkboxError}
```

Comment:
@nickjg231 I was working on the `TextInput` react conversion in this file that has similar logic for the error state (it returns a boolean) but the web component error prop is expecting a string (null if no error) so I am using this logic: 

`error={showError ? signatureError : null}`

Should this line for `va-checkbox` instead be this then?

`error={showError ? checkboxError : null}`

Otherwise va-text-input displays the boolean as the error message (I'm assuming that va-checkbox would as well):

![Screenshot 2023-03-28 at 5 07 19 PM](https://user-images.githubusercontent.com/872479/228377577-7dc80953-76fb-4b90-a51f-c2f63fa1653f.png)